package.json

```diff
@@ -24,11 +26,12 @@
     "stylelint-config-standard-scss": "^13.1.0"
   },
   "lint-staged": {
-    "*.{js,mjs}": "eslint --fix",
-    "*.css": "stylelint --fix"
+    "!docs/**/*.{js,mjs}": "eslint --fix",
+    "*.css": "stylelint --fix --allow-empty-input"
```

Comment:
what does this check do?

Reply:
These flags were to avoid linting the `docs/` folder.